### PR TITLE
Polish translation of cheatsheets page

### PR DESCRIPTION
**Pull request by starsep**
I've added polish translation of cheatsheets page. You can find it [here](http://starsep.github.io/scala.github.com/pl/cheatsheets/).

### CODE REVIEW

**Comment by charafau** [comment]
Thank you, I will make a review as soon as possible.

**Comment by starsep** [comment]
I'll highly appreciate any feedback.
I'm not a translator so many terms might be translated in a rather artificial, clumsy way (maybe leaving English terms would be better?). Neither I am a Scala programmer, I've just started my journey with this language so some concepts might be unclear to me.
Thank you for your quick answer.

**Comment by charafau** [comment]
You should also add link you your translation on other pages, not only on English one.

* /ba/cheatsheets/index.md
* /fr/cheatsheets/index.md
* /ja/cheatsheets/index.md
* /pt-br/cheatsheets/index.md

**Comment by starsep** [comment]
AFAIK there is no need.
I believe that https://github.com/scala/scala.github.com/blob/master/_includes/cheatsheet-sidebar.txt#L31 takes care about proper links.

File: pl/cheatsheets/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+|  `for (x <- xs; y <- ys) yield x*y` _to samo co_ <br>`xs flatMap {x => ys map {y => x*y}}`                  |  zrozumienie pętli for: iloczyn kartezjański |
+|  `for (x <- xs; y <- ys) {`<br>    `println("%d/%d = %.1f".format(x,y, x*y))`<br>`}`                     |  zrozumienie pętli for: imperatywnie<br>[sprintf-style](http://java.sun.com/javase/6/docs/api/java/util/Formatter.html#syntax) |
+|  `for (i <- 1 to 5) {`<br>    `println(i)`<br>`}`                                                        |  zrozumienie pętli for: iterowanie aż do górnej granicy |
+|  `for (i <- 1 until 5) {`<br>    `println(i)`<br>`}`                                                     |  zrozumienie pętli for: iterowanie poniżej górnej granicy |
+|  <span id="pattern_matching" class="h2">pattern matching (dopasowywanie wzorca)</span>                                                         |                 |
+|  <span class="label success">Dobrze</span> `(xs zip ys) map { case (x,y) => x*y }`<br> <span class="label important">Źle</span> `(xs zip ys) map( (x,y) => x*y )` |  używaj słowa kluczowego case w funkcjach w celu dopasowywania wzorca. |
```
Comment (starsep) [inline pl/cheatsheets/index.md]: I'm not sure about this suggestion. Do you want me to replace something with `(pattern matching)`? I've added that phrase at the end.

**Comment by charafau** [comment]
Thanks ! LGTM from me. Dont know if someone else can take a look too. @SethTisue  what do you think?

**Comment by SethTisue** [comment]
thanks Filip & Rafal!

**Comment by mpociecha** [comment]
`zostaje wypisywane` - I'd write `jest wypisywane` or `zostaje wypisane` but:
1. I'm a bit late.
2. It doesn't really matter. :)

**Pull request merged**